modules/UI/videolayout/VideoLayout.js

```diff
@@ -103,9 +103,15 @@ function getPeerContainerResourceId(containerElement) {
 
 const VideoLayout = {
     init(emitter) {
+        this._updateLargeVideoIfDisplayed
+            = this._updateLargeVideoIfDisplayed.bind(this);
+
         eventEmitter = emitter;
 
-        localVideoThumbnail = new LocalVideo(VideoLayout, emitter);
+        localVideoThumbnail = new LocalVideo(
+            VideoLayout,
+            emitter,
+            this._updateLargeVideoIfDisplayed);
```

Comment:
how about binding `this` inline only in this one call that needs it rather than doing it in init?